be/src/exec/pipeline/aggregate/aggregate_streaming_sink_operator.h

```diff
@@ -0,0 +1,49 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+
+#include "exec/pipeline/operator.h"
+#include "exec/vectorized/aggregator.h"
+
+namespace starrocks::pipeline {
+class AggregateStreamingSinkOperator : public Operator {
+public:
+    AggregateStreamingSinkOperator(int32_t id, int32_t plan_node_id, AggregatorPtr aggregator)
+            : Operator(id, "aggregate_streaming_sink_operator", plan_node_id), _aggregator(aggregator) {}
+    ~AggregateStreamingSinkOperator() = default;
+
+    bool has_output() const override { return false; }
+    bool need_input() const override { return true; }
+    bool is_finished() const override;
+    void finish(RuntimeState* state) override;
+
+    Status prepare(RuntimeState* state) override;
+
+    StatusOr<vectorized::ChunkPtr> pull_chunk(RuntimeState* state) override;
+    Status push_chunk(RuntimeState* state, const vectorized::ChunkPtr& chunk) override;
+
+private:
+    Status _push_chunk_by_force_streaming();
```

Comment:
Would better add some comments for these three methods ?

Reply:
fixed